cypress/integration/profile/update.spec.js

```diff
@@ -48,15 +48,15 @@ describe(`profile/update (from login)`, () => {
     goToNextStep('Military Details');
     goToNextStep('Technology');
     cy.findByTestId(MULTI_STEP_SUBMIT_BUTTON).click();
-    cy.wait('@patchUser');
+    // cy.wait('@patchUser');
```

Comment:
comment

Reply:
I will remove this line of code in my next commit as the wait is not needed.

Reply:
I have removed this line of code in my latest commit